content/chapters/software-stack/lecture/slides/software-stack.md

```diff
@@ -0,0 +1,133 @@
+---
+
+## Software Stack
+
+---
+
+![What We Have](media/hardware-software/1-hardware.svg)
+
+----
+
+![What We Want](media/hardware-software/2-features.svg)
+
+----
+
+![Have to Want](media/hardware-software/3-software.svg)
+
```

Comment:
These being learning slides, does it make sense to create "animated" figures as opposed to only having one slide with the last figure?